src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -288,10 +288,19 @@ private bool IsValidFilePath(string path, PSModuleInfo module, bool verifyPathSc
         {
             try
             {
+                // resolve the path so slashes are in the right direction
+                CmdletProviderContext cmdContext = new CmdletProviderContext(this);
+                Collection<PathInfo> pathInfos = SessionState.Path.GetResolvedPSPathFromPSPath(path, cmdContext);
+
+                foreach (PathInfo pathInfo in pathInfos)
```

Comment:
Since wildcards are explicitly being checked and not allowed in the module manifest, is a Debug.Assert sufficient?